### ISSUE TITLE
Cannot find the file.  Error message is incorrect. 

### ISSUE THREAD

**Issue by geraldnolan** [rfennell/AzurePipelines#743]
```bash
##[debug]SYSTEMVSSCONNECTION exists true
##[debug]Found an OAUTH token
##[error]Cannot find the file /home/ssb/myagent/_work/8/s/docs/release-notes.md
##[debug]Processed: ##vso[task.issue type=error;]Cannot find the file /home/ssb/myagent/_work/8/s/docs/release-notes.md
```

I believe the file the code is looking for on line 63 should be ${sourceFile}. [https://github.com/rfennell/AzurePipelines/blob/master/Extensions/WikiUpdater/WikiUpdaterTask/src/GitWikiTask.ts](https://github.com/rfennell/AzurePipelines/blob/master/Extensions/WikiUpdater/WikiUpdaterTask/src/GitWikiTask.ts)

I kept rebuilding trying to change **file/page** with no effect.  After looking at the code, it was clear I was changing the wrong variable.

```javascript
if (dataIsFile === true) {
    if (fs.existsSync(sourceFile)) {
        contents = fs.readFileSync(sourceFile, "utf8");
    } else {
        logError(`Cannot find the file ${filename}`);
        haveData = false;
    }
}
```

**Comment by rfennell** [comment]
Oops, I will sort that now

**Comment by rfennell** [comment]
Fixed in 1.13.2 

Thanks for reporting it
